### DISCUSSION TITLE
Modify request headers

### DISCUSSION THREAD

**Discussion by sjinno** [tauri-apps/tauri#3463] [Q&A]
Hello! My team is adding Tauri to our current webapp in order to create a native desktop app. We use iframes to load the content of 3rd party websites, and we currently modify the request headers through our Chrome extension.

[This stack overflow answer](https://stackoverflow.com/a/27936481) is pretty close to how we modify the request headers, and we were wondering if there would be a way to achieve the modification of headers in Tauri.

I also found a couple of similar questions posted (https://github.com/tauri-apps/tauri/discussions/1145 and https://github.com/tauri-apps/tauri/issues/2975) related to this subject, so I'm wondering if this is work in progress unless the answer/recommendation to #1145 is all there is so far. :)

Thank you very much!

**Comment by jackbuehner** [comment]
See https://github.com/tauri-apps/tauri/discussions/6187

**Comment by FabianLars** [comment]
And this https://github.com/tauri-apps/tauri/discussions/4912#discussioncomment-3406282. It's a bit more extensive than 6187 and also goes into more detail into why it's windows only
